### PR TITLE
Update LDO_Nitehawk-SB_v1.0.cfg

### PR DESCRIPTION
**Pull request by dev-aphex**
alias and override template for the y-axis endstop was missing

### CODE REVIEW

File: user_templates/mcu_defaults/toolhead/LDO_Nitehawk-SB_v1.0.cfg [hunk 1]
```diff
@@ -70,6 +70,18 @@ pin: toolhead:E_FAN
...
+
+## Uncomment the following two lines to configure the Nitehawk adxl sensor
+#[resonance_tester]
+#probe_points:
+## Uncomment below for 250mm build
+#    125, 125, 20
+## Uncomment below for 300mm build
+#    150, 150, 20
+## Uncomment below for 350mm build
+#    175, 175, 20
```
Comment (Frix-x) [inline user_templates/mcu_defaults/toolhead/LDO_Nitehawk-SB_v1.0.cfg:84]: Please remove this part as the resonance testing is defined elsewhere in Klippain

**Review by Frix-x — CHANGES_REQUESTED** [review]
Everthing is almost ok, I just would like to remove the `[resonance_tester]` part that you added

File: user_templates/mcu_defaults/toolhead/LDO_Nitehawk-SB_v1.0.cfg (hunk 1 above)
Comment (dev-aphex) [reply user_templates/mcu_defaults/toolhead/LDO_Nitehawk-SB_v1.0.cfg]: done :)

**Review by Frix-x — APPROVED** [review]
Thanks, I'll merge :)

**Pull request merged**